example-integration/src/main/webapp/WEB-INF/templates/order_details.ftl

```diff
@@ -74,18 +74,44 @@
                 </select>
             </td>
         </tr>
+        <tr>
+            <td>
+                <label for="issuer_id">Issuer ID:</label>
+                <select name="issuerId" id="issuer_id">
+                    <option value="" selected="selected"></option>
+                    <option Value="ABNANL2A">ABN Amro</option>
+                    <option Value="ASNBNL21">ASN Bank</option>
+                    <option value="BUNQNL2A">bunq</option>
+                    <option value="FVLBNL22">F. van Lanschot Bankiers</option>
+                    <option Value="HANDNL2A">Handelsbanken</option>
+                    <option VALUE="INGBNL2A">ING</option>
+                    <option VALUE="KNABNL2H">KNAB</option>
+                    <option Value="RABONL2U">Rabobank</option>
+                    <option Value="RBRBNL21">Regio Bank</option>
+                    <option Value="SNSBNL2A">SNS Bank</option>
+                    <option Value="TRIONL2U">Triodos Bank</option>
+                </select>
```

Comment:
Can we populate this field using the response when requesting the iDEAL issuers?